CWE-94-Python_darwin/dongweiming_Mtime/log.py

```
@@ -0,0 +1,53 @@
+# 添加自定义的客户端ip字段
+d = {'clientip': get_ip_address()}
+
+logger = logger()
+debug = partial(logger.debug, extra=d)
+info = partial(logger.info, extra=d)
+warn = partial(logger.warn, extra=d)
+# error类型的日志记录堆栈
+error = partial(logger.error, exc_info=1, extra=d)
+
+
+def handle_log(socket, address):
```

Ruff (0.13.2)

Unused function argument: `address`

(ARG001)